scripts/onboarding/aks/onboarding-msi-azure-policy/azure-policy.parameters.json

```diff
@@ -94,5 +94,13 @@
         "Microsoft-Perf",
         "Microsoft-RetinaNetworkFlowLogs"
       ]
+    },
```

Comment:
changes looks good to me. Did you validate all these changes in the PR? if not, please validate it them.

Reply:
yes please check int he description with test screenshot